Gems/WhiteBox/Code/Source/Components/WhiteBoxColliderComponent.cpp

```diff
@@ -33,9 +33,8 @@ namespace WhiteBox
         }
     }
 
-    void WhiteBoxColliderComponent::GetProvidedServices(AZ::ComponentDescriptor::DependencyArrayType& provided)
+    void WhiteBoxColliderComponent::GetProvidedServices([[maybe_unused]] AZ::ComponentDescriptor::DependencyArrayType& provided)
```

Comment:
I think you can just get rid of this if it's empty

Reply:
I found it actually easier to go through this code, review it, compare it and modify it when all the functions were present. Unless there are strong feelings against it, for this now I'd keep them.